listenbrainz/webserver/views/art_api.py

```diff
@@ -101,28 +107,61 @@ def cover_art_grid_post():
     if err is not None:
         raise APIBadRequest(err)
 
-    if not isinstance(r["release_mbids"], list):
-        raise APIBadRequest("release_mbids must be a list of strings specifying release_mbids")
-
-    for mbid in r["release_mbids"]:
-        if not is_valid_uuid(mbid):
-            raise APIBadRequest(f"Invalid release_mbid {mbid} specified.")
-
     if "cover_art_size" in r:
         cover_art_size = r["cover_art_size"]
     elif r["image_size"] < 1000:
         cover_art_size = 250
     else:
         cover_art_size = 500
 
-    images = cac.load_images(r["release_mbids"], tile_addrs=tiles, layout=layout, cover_art_size=cover_art_size)
+    # Get release_mbids or release_group_mbids
+    mbids = []
+    entity = "release"
+    
+    def look_for_release_group_mbids():
```

Comment:
I feel this function is named poorly. Right now it sounds more like a function to lookup some data, but its actually a sanity check.  sanity_check_mbids()? Also, a docstring is really needed here.

Reply:
This function assigns values to variables and throws error if the response is not sane. It looks fine but your call!

Docstring added!

Reply:
Done